extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/ignore_explicit_for_joined/IgnoreExplicitForJoinedDefaultValueWithMultiplePUsTest.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.hibernate.orm.ignore_explicit_for_joined;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.arc.Arc;
+import io.quarkus.hibernate.orm.PersistenceUnit;
+import io.quarkus.hibernate.orm.multiplepersistenceunits.model.config.inventory.Plane;
+import io.quarkus.hibernate.orm.multiplepersistenceunits.model.config.user.User;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class IgnoreExplicitForJoinedDefaultValueWithMultiplePUsTest {
+
+    @RegisterExtension
+    static QuarkusUnitTest runner = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClass(User.class)
+                    .addClass(Plane.class)
+                    .addAsResource("application-multiple-pu-discriminator-ignore-explicit-for-joined-default-value.properties",
+                            "application.properties"));
+    @PersistenceUnit("users")
+    @Inject
+    EntityManager emUsers;
+
+    @PersistenceUnit("inventory")
+    @Inject
+    EntityManager emInventory;
+
+    @BeforeEach
+    public void activateRequestContext() {
+        Arc.container().requestContext().activate();
```

Comment:
I think you can simplify this by using `@ActivateRequestContext` in your `@Test`? (@mkouba might confirm if that works)

Reply:
Yep, the `@ActivateRequestContext` should work.

Reply:
Done!